azure-pipelines.yml

```
@@ -48,20 +30,49 @@ variables:
   - name: nugetPackageName
     value: 'nanoFramework.Json'
 
-steps:
+parameters:
+- name: appComponents
```

yamllint

[warning] wrong indentation: expected 2 but found 0

(indentation)